model/VocabularyConfig.php

```diff
@@ -9,6 +9,25 @@ class VocabularyConfig extends BaseConfig
     private $pluginParameters;
     private $languageOrderCache = array();
 
+    const DEFAULT_PROPERTY_ORDER = array("rdf:type", "dc:isReplacedBy",
+    "skos:definition", "skos:broader", "isothes:broaderGeneric",
+    "isothes:broaderPartitive", "isothes:broaderInstantial",
+    "skos:narrower", "isothes:narrowerGeneric", "isothes:narrowerPartitive",
+    "isothes:narrowerInstantial", "skos:related", "skos:altLabel",
+    "skos:note", "skos:scopeNote", "skos:historyNote", "rdfs:comment",
+    "dc11:source", "dc:source", "skosmos:memberOf", "skosmos:memberOfArray");
+
+    const ISO25964_PROPERTY_ORDER = array("rdf:type", "dc:isReplacedBy",
```

Comment:
It would be a good idea to mention this reasoning (like grouping skos:definition together with the other text fields) in the Skosmos wiki, too.